src/platform/windows/events_loop.rs

```diff
@@ -259,12 +283,21 @@ thread_local!(static CONTEXT_STASH: RefCell<Option<ThreadLocalData>> = RefCell::
 struct ThreadLocalData {
     sender: mpsc::Sender<Event>,
     windows: HashMap<winapi::HWND, Arc<Mutex<WindowState>>>,
+    win32_block_loop: Arc<(Mutex<bool>, Condvar)>
+}
+
+fn event_is_resize(event: &Event) -> bool {
```

Comment:
I'd prefer is this function didn't exist. 
`if event_is_resize(event)` can be replaced with `if let Event::WindowEvent{ event: WindowEvent::Resized(..), .. }` and it's just as clear in my opinion.